src/Raven.Client/Data/SubscriptionConfig.cs

```diff
@@ -12,7 +12,7 @@ public class SubscriptionConfig
         public long SubscriptionId { get; set; }
         public SubscriptionCriteria Criteria { get; set; }
         public long? AckEtag { get; set; }
-        public DateTime TimeOfSendingLastBatch { get; set; }
-        public DateTime TimeOfLastClientActivity { get; set; }
+        public long TimeOfSendingLastBatch { get; set; }
```

Comment:
Why is this change? Note that this is client visible value, and this require either the right datatype or explanation how to parse it